src/backend/gl/src/tex.rs

```diff
@@ -384,7 +384,7 @@ pub fn make_without_storage(gl: &gl::Gl, info: &TextureInfo) ->
         _ => return Err(TextureError::UnsupportedSampling),
     }
 
-    set_mipmap_range(gl, target, (0, info.levels));
+    set_mipmap_range(gl, target, (0, info.levels - 1));
```

Comment:
In [Factory.create_texture()](https://github.com/gfx-rs/gfx/blob/master/src/backend/gl/src/factory.rs#L267)